make.ps1

```diff
@@ -113,6 +113,8 @@ elseif ($command -eq "version")
 		{
 			$replacement = (gc $mod) -Replace "Version:.*", ("Version: {0}" -f $version)
 			sc $mod $replacement
+			$replacement = (gc $mod) -Replace "modchooser:.*", ("modchooser: {0}" -f $version)
+			sc $mod $replacement
```

Comment:
Please preemptively change these to `Get-Content` and `Set-Content`, or keep them this way and eventually review https://github.com/OpenRA/OpenRA/pull/9775 once I update it.